### ISSUE TITLE
Competition Time and Distance Restrictions

### ISSUE THREAD

**Issue by FatBoyXPC** [thewca/wca-regulations#359]
The WCA Board seemingly has a time and distance restriction/regulation but it is nowhere in the actual regulations.

**Comment by Laura-O** [comment]
A strict time and distance restriction does not exist and the Board prefers to handle each case separately. This is also mentioned in the Delegate Crash Course.
Therefore, WRC and Board have agreed to not add that to the regulations.

**Comment by FatBoyXPC** [comment]
The reason I bring this up is 1) Not all delegates are even aware of this, and most importantly, 2) Competition organizers who are not delegates certainly are not aware of this.

**Comment by Laura-O** [comment]
1. All delegates should be aware of this as they are made aware of this when they become delegates. The crash course is also mentioned frequently in mails on the delegate list. I am not a friend of multiplying information just because people do not read information that is already available.
2. I am not sure how organizing competitions works in your region, but in general organizers should contact a delegate BEFORE they organize a competition? I don't see the problem here.

**Comment by FatBoyXPC** [comment]
1) How long has this been on the delegates crash course? More importantly: How long has this even existed? I don't recall seeing that when I was given it in January 2015, but that doesn't mean it wasn't there as there was a lot of information to take in. This wouldn't be multiplying information, this would simply be moving it. This move is good, too, because it makes things more transparent. Since this will be a case by case basis, we could use "should not" rather than "must not".

Most importantly: All delegates are **not** aware of this currently. That needs to change. I figured a regulation makes this easiest.

2) Obviously the organizer gets with a delegate before they organize a competition, but often times organizers who have organized successful competitions in the past get into a sort of groove where they know (or more realistically, assume/expect) it will get approved so they go ahead and get the venue (which that's needed before competition approval happens, anyway).

**Comment by SAuroux** [comment]
Here is the regulation you are looking for:

8a) An official WCA competition must:
      8a1) Be approved by the WCA Board.

The delegate crash course just provides some more details on what
competitions might not get an approval. But since this is no fixed rule,
there is nothing that could be put into the regulations/guidelines.

Seriously, there have been mails about this rule and it is in the crash
course since. Further, it should be well-known that the crash course is
updated from time to time. Just recently, I have made a remark about that
when anouncing changes in the crash course on results administration on the
delegates list.

If you as a delegate manage to miss all this, then sorry, you are a bad
delegate. We really cannot babysit everybody.

2016-04-20 12:37 GMT+02:00 James LaChance notifications@github.com:

**Comment by Claster** [comment]
I could see a problem, when an organizer finds a venue and after that comes to a delegate, only to be refused. One always can say "Well, that is a problem of an organizer, as he didn't approach a delegate earlier", however, probably this restriction should be communicated to potential organizers. But I don't think it should be in regulations.

> All delegates are not aware of **this** currently.

What do you mean by "this"? Do you mean, aware of existing of time/distance restrictions? That is not true.

**Comment by Laura-O** [comment]
Yes, it actually would be kind of weird to have a regulation that is rather an instrument for the Board to identify problematic competition requests than a proper guideline. In the discussion about the time/distance "warnings" on the delegate list Tim said that 37% of the competitions in 2015 did not meet this restrictions. I don't have any numbers how many competitions are not approved for these reasons, but I am sure it's not that many.

There's even a tutorial for competition organizers at CubingUSA: [https://www.cubingusa.com/cguide1.php](https://www.cubingusa.com/cguide1.php).
Why not simply add one sentence there?

**Comment by FatBoyXPC** [comment]
@SAuroux: 

It looks like 8a1 could be better phrased as this:

> 8a1) Be approved by the WCA Board, as long as it meets an unknown set of criteria

At least this is more honest and transparent.

> But since this is no fixed rule,
> there is nothing that could be put into the regulations/guidelines.

I completely disagree. The guidelines are full of suggestions, rather than hard rules. This actually seems like the best place for it. I would gladly make an issue over at the guidelines instead if that is what is decided.

> there have been mails about this rule

The only email I remember containing this is the one where Pedro announced the "feature" (though that term is  bit opposite, more like a restriction ;) ) on the website when he completed it. That is most certainly not the right place to announce to all the delegates "hey, we've got this new rule" (I understand it was not new, but it was new to a majority of the people who saw that email).

> If you as a delegate manage to miss all this, then sorry, you are a bad
> delegate. We really cannot babysit everybody.

Given that there was no official announcement of such a rule, delegate is to blame for missing this.

@Claster 

> when an organizer finds a venue and after that comes to a delegate

Yes, this! This has most definitely happened! There is a very common theme of "talk to me when you find a venue" and couple that with an organizer who has organized several competitions before and it's really easy to end up in a situation where somebody already paid for a venue and got their competition denied.

> But I don't think it should be in regulations

Guidelines at the very least. Something that all people can see, not just delegates.

> What do you mean by "this"? Do you mean, aware of existing of time/distance restrictions? That is not true.

Clarification: Not every delegate is aware of this existing rule. This statement is most definitely true.

@Laura-O:

>  it actually would be kind of weird to have a regulation that is rather an instrument for the Board to identify problematic competition requests than a proper guideline

Sensing a theme here about how it should at least exist in the guidelines :)

> Tim said that 37% of the competitions in 2015 did not meet this restrictions

As I stated in that email, I was concerned that this tool would make it easier to deny more competitions based on this rule. I'm curious how many competitions have been denied due to this restriction, as I'm aware of at least 2. There have also been at least 2 that I didn't even give the organizer the attempt to host due to this restriction.

> There's even a tutorial for competition organizers at CubingUSA: https://www.cubingusa.com/cguide1.php.
> Why not simply add one sentence there?

Simply put: It doesn't make sense to be put there. CubingUSA is only relevant to competitions in the US, and this restriction obviously isn't applicable to only the US.

Simply put: This is a rule that is enforced. This is something everybody should be aware about. We have a suggested competition announcement of 4 weeks, and we have an actual regulation for 2 weeks. We don't hide that 4 week guideline from people, nor should we hide this nonsense distance rule.

**Comment by FatBoyXPC** [comment]
I am resurrecting this due to #415 being opened. I feel that if that gets added to the regulations, this is more than appropriate to add to the regulations. The arguments in this thread against it are **not** valid as the Board expresses that unless a competition meets other criteria, you can't host more than 1 competition in 30 days/250km.

Each case is not handled "differently", they are all handled by some sort of guideline they have created. This is no different than competitions being handled differently that are submitted less than a month but greater than 2 weeks.

**Comment by KitClement** [comment]
While this rule is fairly well known among delegates and wouldn't change
much in that regard, I feel like such a regulation (or guideline) would
have utility to those who are not delegates. Organizers may be unaware of
the time/distance guideline, and often look for venues and dates before
approaching delegates with a competition proposal. Having such a
regulation/guideline on the books would at least potentially inform new
organizers that they need to be sure that their competition is not too
close to other competitions, and would provide more transparency in how
competitions are approved by the WCA.

**Comment by Claster** [comment]
To advocate, organizers should contact delegates before securing the venue even without proximity policies. Delegates can have personal reasons not to be availavle at certain dates.

**Comment by FatBoyXPC** [comment]
Looks like #389 takes care of this.
